### PR TITLE
Remove deprecation warnings throughout tutorials

### PR DESCRIPTION
**Pull request by qwadratic**
Skipped tutorial 8, as there is a separate PR #697 fixing it

For #389

### CODE REVIEW

File: examples/zkapps/10-account-updates/src/ProofsOnlyZkApp.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export class ProofsOnlyZkApp extends SmartContract {
 
   deploy(args: DeployArgs) {
     super.deploy(args);
-    this.setPermissions({
+    this.account.permissions.set({
```
Comment (barriebyron) [inline examples/zkapps/10-account-updates/src/ProofsOnlyZkApp.ts:20]: we'll need doc updates for these changes, too.  How do we want to ensure that code changes in examples are reflected in the docs? (Separate PR? so we need an issue for each tutorial? ) 
For example, this code is shown in https://docs.minaprotocol.com/zkapps/tutorials/account-updates#non-upgradeable-proof-only

Comment (LucidSamuel) [reply examples/zkapps/10-account-updates/src/ProofsOnlyZkApp.ts:20]: Yes, I think at the moment we can just create PR's whenever there are code changes that also affect docs.

Comment (qwadratic) [reply examples/zkapps/10-account-updates/src/ProofsOnlyZkApp.ts:20]: @LucidSamuel @barriebyron Not sure, should I create separate PR for content update, or add it in this PR?

Comment (LucidSamuel) [reply examples/zkapps/10-account-updates/src/ProofsOnlyZkApp.ts:20]: I think you can simply add it to this one since we're keeping track of it.

Comment (qwadratic) [reply examples/zkapps/10-account-updates/src/ProofsOnlyZkApp.ts:20]: @LucidSamuel done

**Review by LucidSamuel — APPROVED** [review]
LGTM!

File: examples/zkapps/01-hello-world/src/main.ts [hunk 2]
```diff
@@ -1,17 +1,11 @@
 import { Square } from './Square.js';
 import {
```
Comment (barriebyron) [inline examples/zkapps/01-hello-world/src/main.ts:2]: @halsaphi  @LucidSamuel thanks for the updates.
As discussed in comments, let's do all of the required updates here

This PR is still incomplete. All code changes that are reflected in tutorial docs need testing and updating
https://github.com/o1-labs/docs2/blob/main/docs/zkapps/tutorials/01-hello-world.mdx#L305 
and 
https://github.com/o1-labs/docs2/blob/main/docs/zkapps/tutorials/01-hello-world.mdx#L316 
and 
https://github.com/o1-labs/docs2/blob/main/docs/zkapps/tutorials/01-hello-world.mdx#L330
and so on

Please be sure to run the tutorial after code updates to ensure all is as expected. 

Thank you!

**Review by barriebyron — CHANGES_REQUESTED** [review]
For all code updates, please:
- make the corresponding *.mdx content updates 
- run the tutorial to ensure code changes do not introduce errors

File: examples/zkapps/01-hello-world/src/main.ts (hunk 2 above)
Comment (LucidSamuel) [reply examples/zkapps/01-hello-world/src/main.ts:2]: Thanks for cross-checking @barriebyron, 
I ran the code with my previous PR so isReady wasn't included. 
@qwadratic can you please just update this and also add it to the PR so we could just test and review at once. Thanks :)

Comment (qwadratic) [reply examples/zkapps/01-hello-world/src/main.ts:2]: @LucidSamuel @barriebyron done

**Review by LucidSamuel — APPROVED** [review]
Tested, GTG! Thanks @qwadratic

**Review by LucidSamuel — APPROVED** [review]

**Review by barriebyron — APPROVED** [review]
approving for the doc changes
@LucidSamuel  tested the code changes

There seems to be a CI error that is blocking

**Pull request merged**